doc/release-process.md

```diff
@@ -300,15 +303,15 @@ cat "$VERSION"/*/all.SHA256SUMS.asc > SHA256SUMS.asc
 Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
 Note that all values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
 
-To calculate `m_assumed_blockchain_size`:
-- For `mainnet` -> Take the size of the data directory, excluding `/regtest` and `/testnet3` directories.
-- For `testnet` -> Take the size of the `/testnet3` directory.
+To calculate `m_assumed_blockchain_size`, take the size in GiB of these directories:
+- For `mainnet` -> the data directory, excluding the `/regtest`, `/signet`, and `/testnet3` directories
+- For `testnet` -> `/testnet3`
```

Comment:
We also want to exclude overly large files in the root directory of the network, e.g. a huge debug.log as tripped up in #24418 :smile: I'm not sure what is a good calculation, but maybe a simple reminder would do.

Reply:
Added in reminder in 39f85b4